arch/arm/soc/atmel_sam/sam3x/Kconfig.defconfig.series

```diff
@@ -32,6 +28,12 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 	int
 	default 84000000
 
+config NUM_IRQ_PRIO_BITS
```

Comment:
Since the only SAM3X board is the Arduino Due, lets just remove the !HAS_DTS bits (which means you can remove config NUM_IRQ_PRIO_BITS, config FLASH_BASE_ADDRESS, etc)